applications/GeoMechanicsApplication/custom_geometries/line_interface_geometry.h

```diff
@@ -208,6 +208,26 @@ class LineInterfaceGeometry : public Geometry<Node>
         KRATOS_ERROR << IntegrationSchemeFunctionalityNotImplementedMessage();
     }
 
+    GeometriesArrayType GenerateEdges() const override
+    {
+        const auto points = this->Points();
+
+        // The first edge coincides with the first side of the element
+        auto begin_of_second_side = points.ptr_begin() + (points.size() / 2);
```

Comment:
I'm not sure due to the use, but could this be const auto?

Reply:
As far as I know, iterators are generally not made `const`, since the distinction is in the type of iterator used (`iterator` vs. `const_iterator`). In that sense, it is similar to pointers: we do declare pointers to mutable and immutable objects (i.e. `T*` and `const T*`), but rarely do we make the pointer itself `const` (i.e. `T* const` vs `const T* const`). So I would prefer to keep this `auto` rather than changing this to `const auto`. Is that acceptable to you?

Reply:
Ah yes good point, thanks for clarifying, let's leave it as is!